accounts/views.py

```diff
@@ -1,21 +1,21 @@
 
 from django.urls import reverse_lazy, reverse
 
-from django.views.generic import TemplateView, CreateView, UpdateView, DetailView
+from django.views.generic import TemplateView, CreateView, UpdateView, DetailView, ListView
 from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
 from django.contrib.auth.views import LoginView, LogoutView
 from django.contrib.auth import authenticate, login
 
 from accounts.models import User, Profile

```

Comment:
```suggestion
from .models import User, Profile
```
views.pyにformatがかかってないような気がしますので、きちんと保存してformatかけることを意識してください。